src/snowflake/snowpark/_internal/ast_utils.py

```diff
@@ -0,0 +1,121 @@
+import datetime
+import decimal
+import inspect
+import re
+import sys
+
+from functools import reduce
+from pathlib import Path
+from typing import Any, List, Optional
+
+import snowflake.snowpark._internal.proto.ast_pb2 as proto
+
+NoneType = type(None)
+PYTHON_TO_AST_CONST_MAPPINGS = {
```

Comment:
empty line to separate 13 and 14

Reply:
I'm not a huge fan of `PYTHON_TO_AST_CONST_MAPPINGS`. It's not reused anywhere, and you have complications such as the bytearray case. I would suggest removing this and just adding the cases directly in your one function that uses this global.